docs/releasing.md

```diff
@@ -7,7 +7,50 @@
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
-### Releasing KubeFed
+## Releasing KubeFed
+
+### Automated
+
+Creating a KubeFed release can be automated by running a couple of scripts.
+
+##### Prerequisites
+
+The scripts used to automated the release of KubeFed rely on the following
+prereqs:
+
+- [travis CLI](https://github.com/travis-ci/travis.rb#installation)
```

Comment:
Ideally discovery of the image build status would be accomplished without imposing ruby as a dependency (e.g. via curl). If that's not the case, please ensure the check for the image build status can be optionally disabled. Given that I can check the status of the build trivially via the web interface, I'm not going to install and maintain ruby on my dev machine for the sole purpose of automating this step.

Reply:
I do not have it installed, but I definitely prefer installing a standalone binary to something requiring ruby!